lib/packages/core/src/api/__tests__/apiComposer.spec.ts

```diff
@@ -104,23 +105,29 @@ describe('ApiComposer', () => {
             .withTransactionApi({
                 broadcastTransaction,
                 getTransaction,
-                sendMoney,
+                sendAmount,
```

Comment:
wasn't tested :woman_cartwheeling:

Reply:
Yeah phoenix mobile is still using this...

Reply:
@Titanishu fyi

Reply:
If you update to 0.3.1 (latest) you should change to `sendAmount`.